packages/autocannon-ui-frontend/src/components/ResultSet.jsx

```diff
@@ -36,6 +36,11 @@ const useStyles = makeStyles({
   },
   statHeader: {
     fontWeight: 'bold'
+  },
+  summary: {
+    '&.Mui-expanded': {
+      borderBottom: '1px solid #eaeaea'
```

Comment:
let's avoid hardcoding colors. let's put them in the theme and reference them from there